core/domain/android_validation_constants.py

```diff
@@ -23,14 +23,14 @@
 # changed, just this array has to be modified to include the new
 # interactions or remove existing ones.
 VALID_INTERACTION_IDS = [
-    'Continue', 'EndExploration', 'NumericInput', 'TextInput',
-    'FractionInput', 'NumberWithUnits', 'MultipleChoiceInput',
-    'ItemSelectionInput']
+    'Continue', 'DragAndDropSortInput', 'EndExploration', 'FractionInput',
+    'ItemSelectionInput', 'MultipleChoiceInput', 'NumericInput',
+    'NumberWithUnits', 'TextInput'
+]
 
 SUPPORTED_LANGUAGES = ['en']
 
-INVALID_RTE_COMPONENTS = [
-    'collapsible', 'math', 'skillreview', 'tabs', 'video']
+VALID_RTE_COMPONENTS = ['image', 'link', 'skillreview']
```

Comment:
What is ``link``? We support anchor tags, but I don't think we support any special link tag beyond that.

Reply:
The link RTE component in oppia web refers to the `oppia-noninteractive-link` tag which renders the anchor tag with certain attributes. Here's the template that gets rendered:
https://github.com/oppia/oppia/blob/9cfb5e933388b02ab196f5234e9be40057bc9ca2/extensions/rich_text_components/Link/directives/link.directive.html#L4

Reply:
Ack, thanks. We do not currently support this today in Android. I filed https://github.com/oppia/oppia-android/issues/1126 to track this.

Can you please remove link and skillreview for now since technically neither are supported?

Reply:
@BenHenning I need link and skillreview to publish the lesson...

Reply:
Didn't think about the validation error. :) I suppose we need to keep this then. Thanks.